### ISSUE TITLE
Class 'FUIFirestoreTableViewDataSource' does not conform to protocol 'FUIBatchedArrayDelegate'

### ISSUE THREAD

**Issue by ghowen** [firebase/FirebaseUI-iOS#631]
I am getting this warning for a while, also with the latest version from cocoapods (just updated a sec ago):

> /Users/georg/Projekte/Apps/xxx/Pods/FirebaseUI/Firestore/FirebaseFirestoreUI/FUIFirestoreTableViewDataSource.m:32:17: Class 'FUIFirestoreTableViewDataSource' does not conform to protocol 'FUIBatchedArrayDelegate'

 When I allow the compiler to add stubs, it adds this to FUIFirestoreTableViewDataSource.m:
```
- (void)batchedArray:(nonnull FUIBatchedArray *)array willUpdateWithDiff:(nonnull FUISnapshotArrayDiff<FIRDocumentSnapshot *> *)diff {
    <#code#>
}
```

Here are my current pods:
```
Using Firebase (5.17.0)
Using FirebaseABTesting (2.0.0)
Using FirebaseAnalytics (5.6.0)
Using FirebaseAnalyticsInterop (1.2.0)
Using FirebaseAuth (5.3.1)
Using FirebaseAuthInterop (1.0.0)
Using FirebaseCore (5.3.0)
Using FirebaseDatabase (5.1.0)
Using FirebaseFirestore (1.0.1)
Using FirebaseFunctions (2.2.1)
Using FirebaseInstanceID (3.5.0)
Using FirebaseMessaging (3.3.1)
Using FirebaseRemoteConfig (3.1.0)
Using FirebaseStorage (3.1.0)
Using FirebaseUI (6.1.1)

```

**Comment by morganchen12** [comment]
This should be fixed in #688.

**Comment by morganchen12** [comment]
The fix has been released.
